cmd/snap/cmd_debug_timings_test.go

```diff
@@ -39,6 +39,9 @@ var timingsTests = []timingsCmdArgs{{
 }, {
 	args:  "debug timings --ensure=foo 9",
 	error: "cannot use 'ensure' and change id together",
+}, {
+	args:  "debug timings --ensure=foo --startup=bar",
+	error: "cannot use 'startup' with 'ensure' or change id",
```

Comment:
interestingly you don't have a test for startup + change id :-)

Reply:
Good catch, added.